include/edge_kernels/MomentumEdgePecletAlg.h

```diff
@@ -0,0 +1,49 @@
+// Copyright 2017 National Technology & Engineering Solutions of Sandia, LLC
+// (NTESS), National Renewable Energy Laboratory, University of Texas Austin,
+// Northwest Research Associates. Under the terms of Contract DE-NA0003525
+// with NTESS, the U.S. Government retains certain rights in this software.
+//
+// This software is released under the BSD 3-clause license. See LICENSE file
+// for more details.
+//
+
+#ifndef MOMENTUMEDGEPECLETKERNEL_H_
+#define MOMENTUMEDGEPECLETKERNEL_H_
+
+#include <Algorithm.h>
+#include "stk_mesh/base/Types.hpp"
+#include "PecletFunction.h"
+
+namespace sierra{
+namespace nalu{
+
+class Realm;
+class EquationSystem;
+
+class MomentumEdgePecletAlg: public Algorithm{
+public:
+  // use simd due to ngp_peclet function
+  using DblType = double;
```

Comment:
Comment indicates you wanted to use SIMD here?

Reply:
yeah sorry this is a stale comment.  I'll delete it.